### PR TITLE
Add support for embedding VB source in portable pdb

### PR DESCRIPTION
**Pull request by nguerrera**
VB portion of #12625

### CODE REVIEW

**Comment by nguerrera** [comment]
@tmat @jaredpar @agocke @gafter Please review

**Comment by nguerrera** [comment]
First commit (fixing dup ERRID) is not strictly related and submitted separately as  https://github.com/dotnet/roslyn/pull/13178

File: src/Compilers/Core/MSBuildTaskTests/CscTests.cs [hunk 1]
```diff
@@ -246,5 +246,33 @@ public void SourceLink()
             csc.SourceLink = "";
             Assert.Equal(@"/debug:portable /out:test.exe /sourcelink: test.cs", csc.GenerateResponseFileContents());
         }
+
+        [Fact]
+        public void Embed()
+        {
+            var csc = new Csc();
+            csc.Sources = MSBuildUtil.CreateTaskItems("test.vb");
```
Comment (tmat) [inline src/Compilers/Core/MSBuildTaskTests/CscTests.cs]: >.vb [](start = 59, length = 3)

Should probably be ".cs", not that it functionally matters...

Comment (nguerrera) [inline src/Compilers/Core/MSBuildTaskTests/CscTests.cs]: Fixed.

**Comment by tmat** [comment]
:shipit:

**Comment by nguerrera** [comment]
@dotnet-bot retest windows_debug_unit32_prtest please

retest reason: #13186

**Comment by gafter** [comment]
:shipit:

**Comment by nguerrera** [comment]
@gafter Can this be merged or does it need sign-off from an additional @dotnet/roslyn-compiler reviewer?

**Pull request merged**